src/flynt/cli.py

```diff
@@ -52,6 +52,14 @@ def main():
         default=88,
     )
 
+    parser.add_argument(
+        "-n",
+        "--dry-run",
+        action="store_true",
+        default=False,
+        help="Do not change the file in-place. Useful when running as a linting process."
```

Comment:
Perhaps it's useful to elaborate that it MUST be used with --fail-on-change, otherwise flynt just does nothing and returns nothing or potentially fails.